cpp/Makefile

```diff
@@ -66,6 +66,9 @@ data_install: Makefile data/*.conf data/*.map data/*.ttf data/IxTheo_Notation.cs
 	if [ -d $(INCROND_DIR) ]; then \
 		cp data/pipeline_script_triggers $(INCROND_DIR); \
 	fi
+	@echo "Installing newsletter HTML form"
+	cp data/NewsletterUploadForm.html /usr/local/vufind/public/
+	chcon unconfined_u:object_r:usr_t:s0 /usr/local/vufind/public/NewsletterUploadForm.html
```

Comment:
This should be handled by installer.cc by adding a proper SELinux File Context like in all other cases.

Reply:
Done.